crates/node-bindings/src/parcel/parcel.rs

```diff
@@ -23,7 +22,7 @@ pub struct ParcelNapiBuildResult {}
 pub struct ParcelNapiOptions {
   pub threads: Option<u32>,
   pub node_workers: Option<u32>,
-  pub fs: Option<JsObject>,
+  pub use_file_system: bool,
```

Comment:
`use_rpc_file_system`?

Reply:
In this case, the FileSystem object isn't being passed into Rust from JavaScript. The ParcelV3 public API accepts a FileSystem object which gets mapped in JS to events from the RPC system.

If an FS is supplied to the public options, then we know we are using RPC to delegate calls to the FileSystem so we need to tell the rust-side ParcelNapi to send FS events to JS via RPC.